grpc-api/src/flightTest/java/io/deephaven/grpc_api/flight/FlightMessageRoundTripTest.java

```diff
@@ -0,0 +1,310 @@
+package io.deephaven.grpc_api.flight;
+
+import dagger.BindsInstance;
+import dagger.Component;
+import io.deephaven.db.tables.Table;
+import io.deephaven.db.tables.utils.TableDiff;
+import io.deephaven.db.tables.utils.TableTools;
+import io.deephaven.grpc_api.arrow.FlightServiceGrpcBinding;
+import io.deephaven.grpc_api.auth.AuthContextModule;
+import io.deephaven.grpc_api.barrage.BarrageModule;
+import io.deephaven.grpc_api.session.SessionModule;
+import io.deephaven.grpc_api.session.SessionService;
+import io.deephaven.grpc_api.session.SessionServiceGrpcImpl;
+import io.deephaven.grpc_api.session.SessionState;
+import io.deephaven.grpc_api.util.ExportTicketHelper;
+import io.deephaven.grpc_api.util.Scheduler;
+import io.deephaven.proto.backplane.grpc.HandshakeRequest;
+import io.deephaven.proto.backplane.grpc.HandshakeResponse;
+import io.deephaven.proto.backplane.grpc.SessionServiceGrpc;
+import io.grpc.*;
+import io.grpc.CallOptions;
+import io.grpc.netty.NettyServerBuilder;
+import org.apache.arrow.flight.*;
+import org.apache.arrow.flight.impl.Flight;
+import org.apache.arrow.memory.RootAllocator;
+import org.apache.arrow.vector.VectorSchemaRoot;
+import org.apache.arrow.vector.types.pojo.ArrowType;
+import org.apache.arrow.vector.types.pojo.Field;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import javax.inject.Named;
+import javax.inject.Singleton;
+
+import java.io.IOException;
+import java.util.EnumSet;
+import java.util.Set;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.*;
+
+/**
+ * Deliberately much lower in scope (and running time) than BarrageMessageRoundTripTest, the only purpose
+ * of this test is to verify that we can round trip
+ */
+public class FlightMessageRoundTripTest {
+
+//    @Rule
+//    public GrpcServerRule grpcServerRule = new GrpcServerRule();
```

Comment:
ill do a bit more cleanup in this class before final review, there is a fair bit of dead code in here as long as we can't use the InProcessChannel with barrage inputstreams.